routers/index.js

```diff
@@ -682,9 +682,8 @@ router.post('/secret/shell',urlencodedParser,authenticateToken,function(req, res
    return  res.send("sorry No user defined" + ser)
  }
 else {
-  // const uns = serialize.unserialize(req.body)    // Insecure Deserailiazation => Remote Code Execution
-  // return   res.send(uns)
-  return "data"
+  const uns = serialize.unserialize(req.body)    // Insecure Deserailiazation => Remote Code Execution
```

Comment:
#### Automated PR Comment From coverity
<!-- c12ef8eae13b35be70d95f5d46f1a1a7 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Unsafe deserialization
_High CWE-502_
The application performs deserialization by invoking the dangerous `node-serialize.unserialize()` function. If the application passes untrusted data to this function, an attacker can use specialized payloads to achieve remote code execution (RCE).
### How to fix
Modify the application to use a deserialization function that enforces strict type constraints. The actual function used to replace `node-serialize.unserialize()` will depend on the data being serialized and deserialized.